railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -97,16 +99,20 @@ func UpdateIngress(ctx context.Context, targetContent *file.Content, log logr.Lo
 
 		switch proto := *svc.Protocol; proto {
 		case "tcp":
-			err := UpdateTCPIngress(ctx, log, svc, kiccli, ips)
-			return fmt.Errorf("failed to update tcp ingress. err %v", err)
+			if err := UpdateTCPIngress(ctx, log, svc, kiccli, ips); err != nil {
+				return fmt.Errorf("failed to update tcp ingress. err %v", err)
+			}
 		case "udp":
-			err := UpdateUDPIngress(ctx, log, svc, kiccli, ips)
-			return fmt.Errorf("failed to update udp ingress. err %v", err)
+			if err := UpdateUDPIngress(ctx, log, svc, kiccli, ips); err != nil {
+				return fmt.Errorf("failed to update udp ingress. err %v", err)
+			}
+
 		case "http":
-			err := UpdateIngressV1(ctx, log, svc, cli, ips)
-			return fmt.Errorf("failed to update ingressv1. err %v", err)
+			if err := UpdateIngressV1(ctx, log, svc, cli, ips); err != nil {
+				return fmt.Errorf("failed to update ingressv1. err %v", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to update ingressv1: %w", err)
```

Reply:
this could result in unit test exit code 1. Leave as it is.

Reply:
Also, based on golang doc
If the format specifier includes a %w verb with an error operand, the returned error will implement an Unwrap method returning the operand. And generically, the caller would consume err as an object.

Reply:
How could using the `%w` error wrapping produce an `exit code 1` could you please show me an example of this as I've never seen anything like that?